android/autodispose-androidx-lifecycle/src/androidTest/java/autodispose2/androidx/lifecycle/AndroidLifecycleScopeProviderTest.java

```diff
@@ -22,7 +22,7 @@
 import androidx.lifecycle.Lifecycle;
 import androidx.test.annotation.UiThreadTest;
 import androidx.test.ext.junit.runners.AndroidJUnit4;
-import autodispose2.androidx.lifecycle.test.TestLifecycleOwner;
+import androidx.lifecycle.testing.TestLifecycleOwner;
```

Comment:
I'm not sure if you care about sorting imports, but I would expect this to be sorted above the `androidx.test` imports.

Reply:
thanks, will let spotless clean this up